### PR TITLE
v2.0 release

### PR DESCRIPTION
**Pull request by gabrielbriones**
Signed-off-by: Gabriel Briones <gabriel.briones.sayeg@intel.com>

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 # oneContainer API
 
-OneContainer-API is a platform to enable unified APIs for containerized services and backends in multiple segments like AI, Database, and Media.
```
Comment (unrahul) [inline Readme.md:3]: can u pls update the readme with the missing docs some got deleted..as seen here

File: src/onecontainer_api/async_queue/Dockerfile [hunk 2]
```diff
@@ -3,6 +3,6 @@ FROM python:slim
 WORKDIR /workspace
 
 COPY requirements.txt requirements.txt
-RUN pip install --no-cache-dir -r requirements.txt
```
Comment (unrahul) [inline src/onecontainer_api/async_queue/Dockerfile:6]: nit

File: src/onecontainer_api/async_queue/api.py [hunk 3]
```diff
@@ -1,5 +1,3 @@
-# SPDX-License-Identifier: BSD-3-Clause
```
Comment (unrahul) [inline src/onecontainer_api/async_queue/api.py:1]: let's add licenses to all the files like shown here

File: src/onecontainer_api/backends/dlrs-pytorch/classifier/Dockerfile [hunk 4]
```diff
@@ -2,7 +2,7 @@ FROM sysstacks/dlrs-pytorch-ubuntu:latest
 
 WORKDIR /classifier/
 COPY requirements.txt .
-RUN pip install --no-cache-dir -U -r requirements.txt
+RUN pip install -U -r requirements.txt
```
Comment (unrahul) [inline src/onecontainer_api/backends/dlrs-pytorch/classifier/Dockerfile:5]: nit: --no-cache-dir

Comment (gabrielbriones) [reply src/onecontainer_api/backends/dlrs-pytorch/classifier/Dockerfile:5]: The problem here was those changes were made on github repo but not on the internal repo. Let me fix them.

**Pull request merged**